### ISSUE TITLE
broken on all big-endian architectures

### ISSUE THREAD

**Issue by onlyjob** [lizardfs/lizardfs#264]
LizardFS seems to be broken on all big-endian architectures ever since conversion to Cmake. 
During build on Debian ppc64 I've noticed that the following code in `EnvTests.cmake` is not working properly:

~~~~
TEST_BIG_ENDIAN(BIG_ENDIAN)
if(BIG_ENDIAN)
set(WORDS_BIGENDIAN)
endif()
~~~~

The problem is that `set(WORDS_BIGENDIAN)` never set `cmakedefine WORDS_BIGENDIAN` in `config.h` hence endian sensitive conditionals in `src/common/crc.cc` are not working properly. I fixed this problem with `set(WORDS_BIGENDIAN 1)` in #263.

**Comment by psarna** [comment]
I've prepared gerrit entry for your change, it will be put to review and testing:
http://cr.skytechnology.pl:8081/#/c/1528/

**Comment by onlyjob** [comment]
Great. Thank you, Piotr. Please let me know if you need any help or comments.

**Comment by onlyjob** [comment]
I just tried to register on gerrit with my google account and got the following warning:

> OpenID 2.0 for Google accounts is going away.
Developers should migrate to OpenID Connect by April 20, 2015. [Learn more](https://support.google.com/accounts/answer/6135882).

**Comment by onlyjob** [comment]
How it is going with review? `libcrcutil` is now available from Debian "unstable" so it should be fairly easy to test: https://packages.qa.debian.org/libc/libcrcutil.html

**Comment by onlyjob** [comment]
Thanks. :)

**Comment by onlyjob** [comment]
Although fixed in "master" somehow this fix did not make it into 2.6.0 hence 2.6.0 is still broken on all big-endian architectures because `EnvTests.cmake` never sets `WORDS_BIGENDIAN`...
